jaeger-agent.yaml

```diff
@@ -20,7 +20,7 @@ pipeline:
     with:
       repository: https://github.com/jaegertracing/jaeger
       tag: v${{package.version}}
-      expected-commit: da7a5abb484eccf7f8918d7b875430b337e39cd1
+      expected-commit: 30b2a570773f0d0cc911732727df5dadc3293249
```

Comment:
this looks like a moving tag :alert:

the 1.50 was merged https://github.com/wolfi-dev/os/commit/4468d896a47a21b7cd2c2711e7b9dac2ebecba7f